Mathmagician/Interfaces/ICommand.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Mathmagician
+{
+    interface ICommand
+    {
+        void GenerateNumbers();

```

Comment:
Thanks for the feedback, Kate. Will keep this in mind tonight when I start
plugging away at it again.

On Thu, Aug 4, 2016, 3:13 PM Kate <notifications@github.com> wrote:

> In Mathmagician/Interfaces/ICommand.cs
> <https://github.com/nss-evening-cohort-3/mathmagician-group-project-dr-moneysack-and-friend-s/pull/7#discussion_r73594077>
> :
>
> > @@ -0,0 +1,7 @@
> > +﻿namespace Mathmagician
> > +{
> > +    interface ICommand
> > +    {
> > +        void GenerateNumbers();
>
> If you want to make this a really usable interface, you need to think what
> at barest minimum you want something that is a Command to do. I think in
> your situation, you really want this to create your strings for you to
> input into the console. You could use a method that returns a string in
> your interface. That way you'll know that there will be a method on every
> command that gives you an output, but this really depends on what you'd
> want a Command to do.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/nss-evening-cohort-3/mathmagician-group-project-dr-moneysack-and-friend-s/pull/7/files/7e09d844874c8653a6ecb541525d0a6472e523dd#r73594077>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOmzH7qObzD5YNFV93tmslwLU2Z2Zt98ks5qckfagaJpZM4JcRLz>
> .
>